libs/deno/src/lib.rs

```diff
@@ -62,10 +62,10 @@ pub async fn run(
     );
 
     let flags = args::Flags {
-        subcommand: args::DenoSubcommand::Run(args::RunFlags {
-            script: main_module.path().to_owned(),
-            watch: None,
-        }),
+        // subcommand: args::DenoSubcommand::Run(args::RunFlags {
```

Comment:
weird... want to add a comment or remove it completely? is there a link to an upstream issue open on the deno tracker?

Reply:
I looked through the diffs on the deno repo and it seems they've done some work in the worker/worker factory section of the repo. Unable to locate the issue though (not on their bugtracker either). It also seems broken on 1.39 which I've now changed this PR to be based on.